util/eventValidation.js

```diff
@@ -277,7 +276,8 @@ async function handleValidation(event) {
         }
 
         const violationsByType = new Set(validationErrors.map(err => err.type));
-        for (const [key, value] of Object.entries(mergedTpConfig)) {
+        for (const [key, val] of Object.entries(mergedTpConfig)) {
+            const value = val?.toString()?.toLowerCase();
```

Comment:
Currently spreadsheet gives boolean `true/false` while config-backend api is configured to string `"true"/"false"`. Minor suggestion: what do you think of using boolean everywhere? not right away, some other time